amundsen-kube-helm/templates/helm/amundsen/templates/deployment.yaml

```diff
@@ -7,12 +7,24 @@ spec:
   selector:
     matchLabels:
       run: {{ .Chart.Name }}-{{ .Values.searchServiceName }}
-  replicas: 1
+  replicas: {{ .Values.search.replicas }}
```

Comment:
Could you set a default here?

Reply:
Alright. 

I have the default set in the values file. I think that is a better practice, but it is not a big deal.

Reply:
Yeah, I think overall having it in the values file makes sense too, but, defaulting it here means you don't have to have it in the values file... which is a little more succinct. Anyways, thanks for the change.